From 48ec76438ed52afbcf3a66cf437eae16e57fd695 Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Fri, 15 Dec 2017 19:56:18 +0800
Subject: [PATCH] ARM: dts: rk3288-evb-rk1608: using bb io-domain 1.8v

The rk3288-evb-rk1608 hardware using APIO5_VDD 1.8v,
the default 3.3v bb-iomain cause the abnormal waveform
(<1V @24MHz) on the SPI0 bus (GPIO3_B7~B4), even with max
driving strength 12mA. So we need to switch the bb-iodomain
to 1.8v, then the waveform is corrent.

Change-Id: I0fa49e5348897933ee56867ca458ec3aa519bcba
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-rk1608.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-rk1608.dts b/arch/arm/boot/dts/rk3288-evb-rk1608.dts
index fc4b8a993c38..42fee1b7e109 100644
--- a/arch/arm/boot/dts/rk3288-evb-rk1608.dts
+++ b/arch/arm/boot/dts/rk3288-evb-rk1608.dts
@@ -616,6 +616,7 @@
 &io_domains {
 	status = "okay";
 
+	bb-supply = <&vcc_18>;
 	sdcard-supply = <&vccio_sd>;
 	wifi-supply = <&vcc_18>;
 };
-- 
2.35.3

